github_connector/README.rst

```diff
@@ -179,6 +179,35 @@ Finally, you can download locally the source code of all your branches:
 
    .. image:: https://raw.githubusercontent.com/OCA/interface-github/12.0/github_connector/static/description/github_repository_branch_list.png
 
+
+Download a private repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to fetch the source code from a private repository, you need a token (instead of login/password) with the right scopes:
+
+#. Go to 'Github Account' / 'Settings' / 'Developer settings' / 'Personal access tokens' / 'Generate new token'.
+
+   .. image:: https://raw.githubusercontent.com/OCA/interface-github/12.0/github_connector/static/description/generate_new_token.png
+
+#. In the 'New personal access token' you have to select 'repo scope'.
+
+   .. image:: https://raw.githubusercontent.com/OCA/interface-github/12.0/github_connector/static/description/personal_access_token.png
+
+Scopes:
+
+* **(no scope)** : Grants read-only access to public information (includes public user profile info, public repository info, and gists).
+* **repo** : Grants full access to private and public repositories.
+* **repo:status** : Grants read/write access to public and private repository commit statuses.
+* **repo_deployment** : Grants access to deployment statuses for public and private repositories.
+* **public_repo** : Limits access to public repositories. That includes read/write access to code, commit statuses.
+* **repo:invite** : Grants accept/decline abilities for invitations to collaborate on a repository.
```

Comment:
Do you really need full write access here? And access to accept or decline invitations? Isn't that too much?

Reply:
I tried with everyone else but, but it does not work

https://youtu.be/07F2pHfvHvA

Reply:
That's a clear answer, thanks! :wink:

Reply:
Isn't interesting to put the video link or this explanation for avoiding further questions in the future?

Reply:
Where?